airflow/template/templater.py

```diff
@@ -167,6 +179,8 @@ def render_template(
             return {k: self.render_template(v, context, jinja_env, oids) for k, v in value.items()}
         elif isinstance(value, set):
             return {self.render_template(element, context, jinja_env, oids) for element in value}
+        elif isinstance(value, LiteralValue):
+            return value.value
```

Comment:
I think it is not a good place for this condition because this section marked as `# Fast path for common built-in collections.`

Maybe better move it right after `ResolveMixin`

Reply:
Yeah, that's a better location